lib/install/config/assets.js

```diff
@@ -0,0 +1,40 @@
+// Handles static asset integration - like sass and images
+// Note: You must restart bin/webpack-watcher for changes to take effect
+
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const process = require('process')
+const sharedConfig = require('./shared.js')
+const { devServer } = require('../../package.json')
+
+const production = process.env.NODE_ENV === 'production'
+const hotServerAddr = `http://${devServer.host}:${devServer.port}/`
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.(scss|sass|css)$/i,
+        use: ExtractTextPlugin.extract({
+          fallback: 'style-loader',
+          use: ['css-loader', 'sass-loader']
+        })
+      },
+      {
+        test: /\.(jpeg|png|gif|svg|eot|svg|ttf|woff|woff2)$/i,
```

Comment:
No, we don't have any for now. The assets here will be mostly used within javascripts and css as that's where the templates are. An example with react would be,

```js
import React from 'react'
import ReactDOM from 'react-dom'
import clockIcon from '../counter/images/clock.png';
import './hello-react.sass'

const Hello = props => (
  <div className="hello-react">
    <img src={clockIcon} alt="clock" />
    <p>Hello {props.name}!</p>
  </div>
)

Hello.defaultProps = {
  name: 'David'
}

Hello.propTypes = {
  name: React.PropTypes.string
}

document.addEventListener('DOMContentLoaded', () => {
  ReactDOM.render(
    <Hello name="React" />,
    document.getElementById('react-app'),
  )
})
```
```sass
@font-face
  font-family: 'Lacuna'
  src: url('../fonts/lacuna-webfont.eot')
  src: url('../fonts/lacuna-webfont.eot?#iefix') format('embedded-opentype'),
  url('../fonts/lacuna-webfont.woff') format('woff'),
  url('../fonts/lacuna-webfont.ttf') format('truetype'),
  url('../fonts/lacuna-webfont.svg#Darkenstone') format('svg')
  font-weight: normal
  font-style: normal

  #css-bg-image
    width: 48px
    height: 48px
    background-image: url('../images/clock.png')
    background-repeat: no-repeat
```

Perhaps, we can add an `asset_pack_tag` later?